crates/forge/tests/cli/soldeer.rs

```diff
@@ -1,30 +1,40 @@
 //! Contains various tests related to `forge soldeer`.
 
-use std::{
-    fs::{self, OpenOptions},
-    path::Path,
-};
+use std::{fs, path::Path};
 
 use foundry_test_utils::forgesoldeer;
-use std::io::Write;
 forgesoldeer!(install_dependency, |prj, cmd| {
     let command = "install";
     let dependency = "forge-std~1.8.1";
 
-    let foundry_file = prj.root().join("foundry.toml");
+    let mut foundry_contents = r#"[profile.default]
+src = "src"
+out = "out"
+libs = ["lib"]
 
-    cmd.arg("soldeer").args([command, dependency]).assert_success().stdout_eq(str![[r#"
-🦌 Running Soldeer install 🦌
-No config file found. If you wish to proceed, please select how you want Soldeer to be configured:
-1. Using foundry.toml
-2. Using soldeer.toml
-(Press 1 or 2), default is foundry.toml
-Started HTTP download of forge-std~1.8.1
-Dependency forge-std~1.8.1 downloaded!
-Adding dependency forge-std-1.8.1 to the config file
-The dependency forge-std~1.8.1 was unzipped!
-Writing forge-std~1.8.1 to the lock file.
-Added forge-std~1.8.1 to remappings
+[dependencies]
+
+# See more config options https://github.com/foundry-rs/foundry/blob/master/crates/config/README.md#all-options
+"#;
+    let foundry_file = prj.root().join("foundry.toml");
+    fs::write(&foundry_file, foundry_contents).unwrap();
+
+    // force use special symbols on windows
+    cmd.env("WT_SESSION", "true");
+    cmd.arg("soldeer").args([command, dependency]).assert_success().stderr_eq(str![[r#"
+┌  🦌 Soldeer Install 🦌
+│
+◆  Done reading config
+│  
+[6A[1A
+[2K[1B[1A◇  Installing forge-std~1.8.1
+[6B◆  Dependency added to config
+│  
+◆  Dependency added to lockfile
+│  
+◆  Dependency added to remappings
```

Comment:
looks like we need to escape ◇ here

Reply:
Removing the weird characters before the diamonds also works, but yeah probably easier to just wildcard it